web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -77,7 +77,11 @@ export class LayerDialogComponent implements OnDestroy {
     const layer = {
       id: this.layerId,
     };
-    this.dataStoreService.updateProjectLayer(this.projectId, layer);
+    try {
```

Comment:
This function won't throw an exception, it'll return a Promise that fails with error.

Also, I assume we'll want to close the dialog and let the user know the layer was saved on completion? (feel free to add a `// TODO: ....` for now for that part).

Reply:
thank you for catching this :)

Reply:
Very funny :)